### PR TITLE
Filter shared durable inactive subscriptions when displaying from UI

### PR DESCRIPTION
**Pull request by akalankapagoda**
When displaying shared durable subscriptions, display only active subscriptions. If there are no active shared subscriptions for a particular subscriber ID, then display only one inactive shared durable subscription for the same.

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java [hunk 1]
```diff
@@ -25,10 +25,7 @@
 import org.wso2.andes.server.management.AMQManagedObject;
 
 import javax.management.NotCompliantMBeanException;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```
Comment (sajinidesilva) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java]: Should not import with *

File: modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java [hunk 2]
```diff
@@ -115,20 +112,43 @@ public String getObjectInstanceName() {
...
                     }
 
-                    allSubscriptionsForTopics.add(renderSubscriptionForUI(s,pendingMessageCount.intValue()));
+                    subscriptionsToDisplay.add(s);
+                }
+
+                // In UI only one inactive shared subscription should be shown if there are no active subscriptions.
+                for (Map.Entry<String, AndesSubscription> inactiveEntry : inactiveSubscriptions.entrySet()) {
+                    // If there are active subscriptions with same target queue, we skip adding inactive subscriptions
+                    if (!uniqueSubscriptionIDs.contains(inactiveEntry.getKey())) {
```
Comment (sajinidesilva) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java]: Change it like !(uniqueSubscriptionIDs.contains(inactiveEntry.getKey())) for more readability

File: modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java (hunk 1 above)
Comment (akalankapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java]: Will update.

File: modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java (hunk 2 above)
Comment (akalankapagoda) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/information/management/SubscriptionManagementInformationMBean.java]: Will update.

**Pull request merged**